Improve AppStream metainfo

zturtleman opened the pull request:
- Correct the software license to be GPL-2.0-or-later.
- Remove the hyphen in the developer name.
- Add social-audio intense due to unmoderated voice chat.
- Some syntax / order changes to match ioquake3's metainfo.

Mailaender commented on com.q3rally.Q3Rally.yaml:
```diff
@@ -28,14 +28,15 @@ modules:
-      - type: patch
-        path: appstream.patch
+      - type: file
+        path: com.q3rally.Q3Rally.metainfo.xml
```
Can you please include the second patch here?

zturtleman replied:
Okay, I added it as a second patch.

zturtleman committed:
Improve AppStream metainfo

- Correct the software license to be GPL-2.0-or-later.
- Remove the hyphen in the developer name.
- Add social-audio intense due to unmoderated voice chat.
- Some syntax / order changes to match ioquake3's metainfo.
